install.md

```diff
@@ -18,3 +18,9 @@ Optionally set `TELEMETRY_CA_CERT` to a URI serving PEM for a CA cert
 to use when validating Telemetry requests.
 
 Test by using `oc logs -f <podname>`to make sure it's running.
+
+## Expose the route
+```
+oc expose service/cluster-support-bot
```

Comment:
I tried this, and it created an insecure route.  Seems like we need something like:

```sh
oc create route edge --service=cluster-support-bot
```

to get a secure route.

Reply:
Updated!